### DISCUSSION TITLE
Generating blood reports

### DISCUSSION THREAD

**Discussion by BioinfoGuy10** [synthetichealth/synthea#1486] [General]
Hello,
I am new to synthea and struggling a bit to navigate using it.
I am looking to generate blood reports for normal patients, diabetic patients, cancer patients and alzheimers patients.
Is there a way I can generate about a 1000 reports for each?

**Comment by Martinfacot** [comment]
Hi,
I'm also interested in doing something similar. Have you found a way to do it?

**Comment by jawalonoski** [comment]
You can run `./gradlew concepts` to get a list of terminology codes that Synthea supports out of the box.

The data will be written to `./output/concepts.csv`.

If you see a code that matches what you're looking for when you say "_blood report_" then it is possible that Synthea could generate it. If it doesn't, then that is something that would have to be added to a module.

**Reply by Martinfacot** [reply]
Thank you for your prompt reply!
